dockerfiles/e2e/docker-entrypoint.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
```

Comment:
this script is not passing at https://www.shellcheck.net/
```
$ shellcheck myscript
 
Line 3:
if [ -z $TS_SELENIUM_BASE_URL ]; then
        ^-- SC2086: Double quote to prevent globbing and word splitting.

Did you mean: (apply this, apply all SC2086)
if [ -z "$TS_SELENIUM_BASE_URL" ]; then
 
Line 18:
        cd local_tests
        ^-- SC2164: Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

Did you mean: (apply this, apply all SC2164)
        cd local_tests || exit
 
Line 22:
        cd e2e
        ^-- SC2164: Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

Did you mean: (apply this, apply all SC2164)
        cd e2e || exit

$ 
```